crypto/ct/ct_log.c

```diff
@@ -243,26 +243,24 @@ int CTLOG_STORE_load_file(CTLOG_STORE *store, const char *file)
     if (load_ctx->conf == NULL)
         goto end;
 
-    ret = NCONF_load(load_ctx->conf, file, NULL);
-    if (ret <= 0) {
+    if (!NCONF_load(load_ctx->conf, file, NULL)) {
```

Comment:
I looked into the code for NCONF_load (and the chain of function calls it ultimately gets its return value from) and I only ever saw 1 or 0. However, if the rest of the code is using "<= 0", is it better to stick with that for consistency?